src/d3a/models/strategy/settlement_strategies/__init__.py

```diff
@@ -0,0 +1,142 @@
+from d3a_interface.constants_limits import ConstSettings
+from pendulum import duration
+
+from d3a.constants import SettlementTemplateStrategiesConstants
+from d3a.d3a_core.exceptions import MarketException
+from d3a.models.strategy.update_frequency import TemplateStrategyBidUpdater, \
+    TemplateStrategyOfferUpdater
+
+
+def settlement_market_strategy_factory(
```

Comment:
I do not really understand the advantage of having a factory class vs the factory method that is implemented here. inline methods for validation are not really needed here, since the selection of the class is defined by a single constant (ENABLE_SETTLEMENT_MARKETS) and does not require extensive validation which could potentially justify changing the factory method to a factory class. Furthermore, if the need appears at some point in the future it can easily be converted to a class, but for now it is overengineering.